PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-applicationinsights/src/main/java/com/microsoft/azure/toolkit/intellij/applicationinsights/connection/ApplicationInsightsResourceDefinition.java

```diff
@@ -34,11 +30,10 @@
 
 @Getter
 public class ApplicationInsightsResourceDefinition extends AzureServiceResource.Definition<ApplicationInsight> implements IJavaAgentSupported {
-    public static final ApplicationInsightsResourceDefinition INSTANCE =
-            new ApplicationInsightsResourceDefinition("Azure.Insights", "Azure Application Insights", AzureIcons.ApplicationInsights.MODULE.getIconPath());
+    public static final ApplicationInsightsResourceDefinition INSTANCE = new ApplicationInsightsResourceDefinition();
```

Comment:
It seems that we could remove the `INSTANCE`, which could be replaced by name directly